.github/actions/install-chrome/action.yml

```
@@ -0,0 +1,16 @@
+name: Install Chrome browser and driver for Testing
+description: Download and install the latest available Chrome for Testing and Chromedriver
+
+runs:
+  using: composite
+  steps:
+    - id: install-chrome
+      name: Install Chrome
+      shell: bash
+      run: |
+        sudo apt-get remove google-chrome-stable
+        npx @puppeteer/browsers install chrome@stable
+        npx @puppeteer/browsers install chromedriver@stable
```

I think we can omit `stable` here, it should be the default.
```suggestion
        npx @puppeteer/browsers install chrome
        npx @puppeteer/browsers install chromedriver
```